backend/tests/e2e/test_api_e2e.py

```
@@ -124,7 +124,7 @@ async def realistic_llm_generate_response(messages, session_id, stage_execution_
             
             # Import the hook context
             from tarsy.hooks.typed_context import llm_interaction_context
-            from tarsy.models.llm import LLMMessage
+            from tarsy.models.unified_interactions import LLMMessage
```

Ruff (0.12.2)

`tarsy.models.unified_interactions.LLMMessage` imported but unused

Remove unused import: `tarsy.models.unified_interactions.LLMMessage`

(F401)